docs/contribute.md

```diff
@@ -0,0 +1,38 @@
+# Contribution Guidelines
+
+The **RPM Build Pipeline** is open source → see our [LICENSE](LICENSE).
```

Comment:
The `LICENCE` file is missing.

Reply:
I'm going to submit this in a separate PR;  should be Apache 2.0, but waiting for a formal ACK from the legal team.

Reply:
I opened #12  for this.